web/blog/tenzir-v4.1/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Tenzir v4.1
+authors: [dominiklohmann]
+date: 2023-08-31
+tags: [release, pipelines, operators]
+---
+
+After our successful launch of app.tenzir.com of Tenzir v4.0 at Black Hat, [the
+new v4.1 release][github-release] continues with several enhancements based on
+early feedback. We bring to you a (i) new mechanism to pause pipelines, (ii) a
+new operator to match Sigma rules, (iii) new operators for in-pipeline
+(de)compression, and (iv) a revamp of the `show` operator.
+
+[github-release]: https://github.com/tenzir/vast/releases/tag/v4.1.0
+
+![Tenzir v4.1](tenzir-v4.1.excalidraw.svg)
+
+<!--truncate-->
+
+## Pausing Pipelines
+
+Tenzir now supports pausing pipelines. Previously, users were able to deploy and
+start pipelines, and to let them run until they completed (or failed) or
+manually stopped. With new paused state, we now have the following five possible
+states of a pipeline:
+
+- *Created*: The pipeline was created, but never started
+- *Running*: The pipeline is processing data
+- *Paused*: A running pipeline has been suspended but still retains its
+  in-memory state
+- *Stopped*: A pipeline is not running and has no in-memory state
+- *Failed*: The pipeline stopped unexpectedly
+
+Pausing suspends a pipeline in an instant, causing its execution to stop. When a
+paused pipeline is started, it resumes right where it left off as opposed to
+restarting all the way from the beginning.
```

Comment:
Reading this again, I immediately asked myself about delivery guarantees, as "where it left off" potentially implies strong guarantees. I'm not sure how to alleviate this, just pointing it out.

Reply:
I don't know how to alleviate this either, but I think this blog post does not promise to be as technically accurate as possible, so it shouldn't matter much.